lambda_layers/core_layer/python/core_layer/helper.py

```diff
@@ -7,25 +7,25 @@
 from botocore.exceptions import ClientError
 from sqlalchemy import func
 
-from core_layer.connection_handler import get_db_session
+is_test = 'TESTMODE' not in os.environ
```

Comment:
If I'm not getting it totally wrong, then this is super counterintuitive.

If the `TESTMODE` variable is set. then is_test is `false`?
=> Setting the TESTMODE variable enables the production code? 

I find that quite misleading.

`is_test = 'TESTMODE' in os.environ and os.environ['TESTMODE']=1` would be better suited imho.

Reply:
Notable objection. 
I followed a recommendation and discussion to implement it this way, maybe this is not the best.
If we change it `is_test = 'TESTMODE' in os.environ and os.environ['TESTMODE']=1` we also have to set the TESTMODE in every test file.

To be more intuitive could we do it the other way round?
Set new environment variable `"PRODMODE": 1`.
`is_test = 'PRODMODE' not in os.environ`

Reply:
Why not only?
`is_test = 'TESTMODE' in os.environ`

The value of the environment variable doesn't matter, existence should be enough. In my opinion.

Reply:
@ChristianBay but we are setting the variable only in production, so the name is misleading. 
I like @Th3da's suggestion to just rename the variabel to `PRODMODE` and only check for existence.

Reply:
Ah, I got it. Thanks.
PRODMODE is fine for me.

Reply:
PRODMODE may be still a little bit misleading because it's not clearly distinguished from STAGE with values "prod", "dev", "qa".
What do you think about this proposal from @PotSchmoell: DEPLOYMENT_MODE ? 
We could set `DEPLOYMENT_MODE="pipeline"` and only check for existence.
So there is no overlap to any meaning of STAGE.